indico/modules/auth/controllers.py

```diff
@@ -212,6 +213,9 @@ def _create_identity(self):
         identity = Identity(user=self.user, provider=self.identity_info['provider'],
                             identifier=self.identity_info['identifier'], data=self.identity_info['data'],
                             multipass_data=self.identity_info['multipass_data'])
+        self.user.log(UserLogRealm.user, LogKind.positive, 'Accounts', 'External account linked',
+                      data={'Provider': multipass.identity_providers[identity.provider].title,
+                            'Identifier': identity.identifier})
```

Comment:
Can this identifier be a uuid?

Reply:
no, uuid identifiers are only used for local accounts when usernames are disabled